fvm/bootstrap.go

```diff
@@ -335,8 +335,34 @@ func (b *bootstrapExecutor) Preprocess() error {
 }
 
 func (b *bootstrapExecutor) Execute() error {
-	if b.rootBlock == nil {
-		b.rootBlock = flow.Genesis(b.ctx.Chain.ChainID()).ToHeader()
+	if b.rootHeader == nil {
+		// create the raw content for the genesis block
+		payload := flow.NewEmptyPayload()
```

Comment:
looks good, but is there any particular reason not to keep this logic in the `model/flow` package